src/coffee/bootstrap-tour.coffee

```diff
@@ -473,9 +473,34 @@
         .on "#{@_reflexEvent(step.reflex)}.tour-#{@_options.name}", =>
           if @_isLast() then @next() else @end()
 
+      limits = {
+        top:$(window).scrollTop()
+        bottom:$(window).scrollTop() + $(window).height()
+        left:$(window).scrollLeft()
+        right:$(window).scrollLeft() + $(window).width()
+      }
+
       $element
       .popover(
-        placement: step.placement
+        placement: (popover,element)->
```

Comment:
I wonder if that logic will be useful for another purpose. If not, I guess it's ok here, or do you suggest us to create another function for this ?